tests/cases/fourslash/refactorConvertStringOrTemplateLiteral_escapeSequences.ts

```diff
@@ -94,3 +94,15 @@ edit.applyRefactor({
     // newContent is: let s = `\u0041\u0061${text}\0\u0000`;
     newContent: 'let s = `\\u0041\\u0061${text}\\0\\u0000`;'
 });
+
+// @Filename: /i.ts
+////let s = /*i1*/'$`' + text + "`\\"/*i2*/;
+
+goTo.select("i1", "i2");
+edit.applyRefactor({
+    refactorName: "Convert to template string",
+    actionName: "Convert to template string",
+    actionDescription: ts.Diagnostics.Convert_to_template_string.message,
+    // newContent is: let s = `\$\`${text}\`\\`;
+    newContent: 'let s = `\\$\\`${text}\\`\\\\`;'
```

Comment:
There's no reason for the `$` to be escaped here, since it isn't immediately followed by a `{`.

Reply:
@rbuckton: yeah, that was an intentional bit in the last PR: I think that people prefer to see escaped `$`s to make things clear, no?  (Especially in something confusing like `$${n}`.)  If you prefer the minimal route, then maybe I'll do that in a separate PR?  (The change is easy, and I have the minimal thing in a comment -- look for the "less-backslash-happy version" comment.)

Reply:
I'd prefer the unescaped `$`, but since this isn't directly being used as the template for a tagged template expression, its probably fine? Again, I'm unsure whether this would affect anyone using custom transforms where they *wouldn't* want the extra `\`.